scripts/version_tag.sh

```diff
@@ -74,24 +81,32 @@ echo "Updating version...";
 gulp bump --type=$TYPE
 
 # Get the new package version.
-PACKAGE_VERSION=$(cat package.json \
+SEMANTIC_PACKAGE_VERSION=$(cat package.json \
```

Comment:
type=prerelease isn't in the documented usage on top

Why the -0 in the package.json version? Related: why alpha0 and not alpha1? I have never seen a package anywhere with version alpha0.

Also for gulp bump, that seems to be the normal behaviour. But we could say if prerelease is given to this script, we first do a gulp bump minor and then a prerelease bump?

Reply:
I didn't document it at the top to try preventing users from doing this manually, seeing as we will only update the style guide in minor versions and possible major versions from now on.

For the -0 in the package.json version... yes this seems to be a gulp bump issue with --type=prerelease.

If we try doing a minor release and then a prerelease it goes as follows:

Say we are on 2.9.0
We gulp bump --type=minor --> 2.10.0 is the result
We then gulp bump --type=prerelease --> 2.10.1-0 is the result

So that doesn't solve this either I think... 

@daften @Jeroen005 @delrueba Any ideas on how we could fix this?